src/website/app/demos/Card/examples/card-title/withMetaInfo.js

```diff
@@ -24,17 +24,18 @@ export default {
   title: 'With meta information',
   // $FlowFixMe
   backgroundColor: mineralTheme.color_gray_10,
-  description:
-    'If you provide meta information, such as a date or category, it will display above the title, which will automatically display in the minor style.',
+  description: `When you must provide information that doesn't belong to every Card in a set, supply it as \`meta\` information.
+The information will display above the title, which will automatically display in the minor style.
+If information is in every card in the set, consider using the \`subtitle\` attribute.`,
   scope: { Card, CardBlock, CardTitle, DemoLayout },
   source: `
     <DemoLayout>
       <Card>
-        <CardTitle meta="July 24, 2017">Card title</CardTitle>
+        <CardTitle meta="Meta information for this Card">Card Title</CardTitle>
```

Comment:
The casing here seems odd. Maybe just title case it?

Reply:
addressed